pipeline/src/resources/ColorAnnotator.py

```diff
@@ -0,0 +1,59 @@
+
+import tornado.web
+import socket
+import tornado.ioloop
+import json
+
+
+class Annotator(tornado.web.RequestHandler):
+    def __init__(self):
+        self._addr = ''
+        self._port = 12345
```

Comment:
FYI: I set the port to be 8080. I created #19 to move these magic numbers into config files.

Reply:
Addressed in #20